runtime/common/src/crowdloan/migration.rs

```diff
@@ -121,6 +164,25 @@ pub mod crowdloan_index_migration {
 				total_balance >= fund.raised,
 				"Total balance in new account is different than the funds raised."
 			);
+
+			let leases = Leases::<T>::get(para_id).unwrap_or_default();
+			let mut new_account_found = false;
+			for maybe_deposit in leases.iter() {
+				if let Some((who, _amount)) = maybe_deposit {
+					if *who == old_fund_account {
+						panic!("Old fund account found after migration!");
```

Comment:
```suggestion
						log::error!(target: "runtime", "Old fund account found after migration!");
```

Reply:
Why shouldnt i panic in try runtime?

Reply:
ahh sorry I didn't see it is try-runtime, panic all you want :p